tests/factory_fixtures/notes.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from models.notes import NotesModel
+
+@pytest.fixture
+def note_attributes():
+    def _note_attributes(text, user, ticket):
+        return {
+            'text': text,
+            'user': user.id,
+            'ticketid': ticket.id
+        }
+    yield _note_attributes
+
+@pytest.fixture
+def create_note(note_attributes, create_join_staff):
```

Comment:
These two factories here don't seem to be in use. Is there a reason you decided to not use them, and instead create Fakes?

Reply:
I initially did this to avoid creating a ticket fixture, since someone else was working on that.